functional-tests.sh

```diff
@@ -1028,6 +1028,7 @@ function test_bucket_replication() {
 			break
 		fi
 		assert_success "$start_time" "${FUNCNAME[0]}" mc_cmd cp "$FILE_0_B" "${SERVER_ALIAS}/${bkt1}/${object_name}"
+		sleep 5s # to make sure objects reflects
```

Comment:
Looks more like you are trying to hide an underlying issue.

Reply:
in local testing and even with play it always worked. only during CI while mint tests sometimes it fails.

Reply:
@shtripat A wait is never the solution. It may make it less likely - but AFAICT this shouldn't be needed in the first place, unless we are not consistent.

Reply:
I agree, but not sure as some cases it just fails immediate rm for the object which was uploaded using `mc cp`.
This test anyway was added for a scenario which in itself is very rare. May be we can just disable this test for the time being.

Reply:
We should figure out why it is failing in the first place.